src/app/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -1,6 +1,13 @@
 import { Component, Inject, OnInit } from '@angular/core';
 import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material';
 
+export class ConfirmationDialogData {
```

Comment:
Just curious, why did you move this? Convention, or is something enforcing that exports be at the top? (I actually prefer it this way).

Reply:
Cannot be used before declared or something like that.